pkg/arvo/app/acme.hoon

```diff
@@ -584,10 +584,11 @@
       ~|  [%no-next-domain idx=idx]
       (head (skim pending |=([turf idx=@ud ?] =(idx ^idx))))
     ::  XX should confirm that :turf points to us
-    ::  confirms that domain exists (and an urbit is on :80)
```

Comment:
This still depends on the port, but now uses `80` or `443` depending on whether we're secure or not, right? So, maybe this should say `(and an urbit is on the standard port)` or similar.